From mfm@muteddisk.com  Tue Oct  5 15:33:13 2010
From: matt mooney <mfm@muteddisk.com>
To: kernel-janitors@vger.kernel.org
Cc: Greg Kroah-Hartman <gregkh@suse.de>,
	David Brownell <dbrownell@users.sourceforge.net>,
	Felipe Balbi <felipe.balbi@nokia.com>,
	Matthew Dharm <mdharm-usb@one-eyed-alien.net>,
	David Vrabel <david.vrabel@csr.com>, linux-usb@vger.kernel.org,
	linux-kernel@vger.kernel.org, usb-storage@lists.one-eyed-alien.net
Subject: usb: change to new flag variable
Date: Fri, 24 Sep 2010 12:17:33 -0700
Message-Id: <1dd0f4a1709a9b139fe87f2b311cc0d129900b3f.1285355033.git.mfm@muteddisk.com>

Replace EXTRA_CFLAGS with ccflags-y.

Signed-off-by: matt mooney <mfm@muteddisk.com>
Acked-by: WANG Cong <xiyou.wangcong@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/atm/Makefile      |    4 +---
 drivers/usb/core/Makefile     |    4 +---
 drivers/usb/gadget/Makefile   |    4 +---
 drivers/usb/host/Makefile     |    4 +---
 drivers/usb/misc/Makefile     |    4 +---
 drivers/usb/musb/Makefile     |    4 +---
 drivers/usb/storage/Makefile  |    2 +-
 drivers/usb/wusbcore/Makefile |    4 +---
 8 files changed, 8 insertions(+), 22 deletions(-)

--- a/drivers/usb/atm/Makefile
+++ b/drivers/usb/atm/Makefile
@@ -8,6 +8,4 @@ obj-$(CONFIG_USB_UEAGLEATM)	+= ueagle-at
 obj-$(CONFIG_USB_ATM)		+= usbatm.o
 obj-$(CONFIG_USB_XUSBATM)	+= xusbatm.o
 
-ifeq ($(CONFIG_USB_DEBUG),y)
-EXTRA_CFLAGS += -DDEBUG
-endif
+ccflags-$(CONFIG_USB_DEBUG)	:= -DDEBUG
--- a/drivers/usb/core/Makefile
+++ b/drivers/usb/core/Makefile
@@ -16,6 +16,4 @@ endif
 
 obj-$(CONFIG_USB)	+= usbcore.o
 
-ifeq ($(CONFIG_USB_DEBUG),y)
-EXTRA_CFLAGS += -DDEBUG
-endif
+ccflags-$(CONFIG_USB_DEBUG) := -DDEBUG
--- a/drivers/usb/gadget/Makefile
+++ b/drivers/usb/gadget/Makefile
@@ -1,9 +1,7 @@
 #
 # USB peripheral controller drivers
 #
-ifeq ($(CONFIG_USB_GADGET_DEBUG),y)
-	EXTRA_CFLAGS		+= -DDEBUG
-endif
+ccflags-$(CONFIG_USB_GADGET_DEBUG)	:= -DDEBUG
 
 obj-$(CONFIG_USB_DUMMY_HCD)	+= dummy_hcd.o
 obj-$(CONFIG_USB_NET2280)	+= net2280.o
--- a/drivers/usb/host/Makefile
+++ b/drivers/usb/host/Makefile
@@ -2,9 +2,7 @@
 # Makefile for USB Host Controller Drivers
 #
 
-ifeq ($(CONFIG_USB_DEBUG),y)
-	EXTRA_CFLAGS		+= -DDEBUG
-endif
+ccflags-$(CONFIG_USB_DEBUG)	:= -DDEBUG
 
 isp1760-objs := isp1760-hcd.o isp1760-if.o
 fhci-objs := fhci-hcd.o fhci-hub.o fhci-q.o fhci-mem.o \
--- a/drivers/usb/misc/Makefile
+++ b/drivers/usb/misc/Makefile
@@ -26,6 +26,4 @@ obj-$(CONFIG_USB_YUREX)		+= yurex.o
 
 obj-$(CONFIG_USB_SISUSBVGA)	+= sisusbvga/
 
-ifeq ($(CONFIG_USB_DEBUG),y)
-EXTRA_CFLAGS += -DDEBUG
-endif
+ccflags-$(CONFIG_USB_DEBUG)	:= -DDEBUG
--- a/drivers/usb/musb/Makefile
+++ b/drivers/usb/musb/Makefile
@@ -78,6 +78,4 @@ endif
 
 # Debugging
 
-ifeq ($(CONFIG_USB_MUSB_DEBUG),y)
-	EXTRA_CFLAGS += -DDEBUG
-endif
+ccflags-$(CONFIG_USB_MUSB_DEBUG) := -DDEBUG
--- a/drivers/usb/storage/Makefile
+++ b/drivers/usb/storage/Makefile
@@ -5,7 +5,7 @@
 # Rewritten to use lists instead of if-statements.
 #
 
-EXTRA_CFLAGS	:= -Idrivers/scsi
+ccflags-y	:= -Idrivers/scsi
 
 obj-$(CONFIG_USB_UAS)		+= uas.o
 obj-$(CONFIG_USB_STORAGE)	+= usb-storage.o
--- a/drivers/usb/wusbcore/Makefile
+++ b/drivers/usb/wusbcore/Makefile
@@ -21,6 +21,4 @@ wusb-wa-objs :=	wa-hc.o		\
 		wa-rpipe.o	\
 		wa-xfer.o
 
-ifeq ($(CONFIG_USB_WUSB_CBAF_DEBUG),y)
-EXTRA_CFLAGS += -DDEBUG
-endif
+ccflags-$(CONFIG_USB_WUSB_CBAF_DEBUG) := -DDEBUG
